src/sentry/api/endpoints/issue_occurrence.py

```diff
@@ -41,8 +44,28 @@ def post(self, request: Request) -> Response:
         Write issue occurrence and event data to a Kafka topic
         ``````````````````````````````````````````````````````
         :auth: superuser required
+        :pparam: string dummy: pass 'True' to load a dummy event instead of providing one in the request
         """
-        event = request.data.pop("event")
+        event = {}
+        if request.query_params.get("dummy") == "True":
+            event = {
+                "event_id": "44f1419e73884cd2b45c79918f4b6dc4",
+                "project_id": 1,
+                "title": "This is bad",
+                "platform": "python",
+                "tags": {"environment": "prod"},
+                "timestamp": ensure_aware(datetime.now()),
+                "message_timestamp": ensure_aware(datetime.now()),
+            }
+        else:
+            event = request.data.pop("event", None)
+
+        if not event:
```

Comment:
Could also load the fake event if no event is passed? But that might be unexpected behavior.

Reply:
yeah I would skip and keep as is